engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -145,6 +192,55 @@ public GroovyDeephavenSession(
         }
     }
 
+    /**
+     * Adds the default imports that Groovy users assume to be present.
+     */
+    private void addDefaultImports(ImportCustomizer imports) {
+        // TODO (core#230): Remove large list of manual text-based consoleImports
+        // NOTE: Don't add to this list without a compelling reason!!! Use the user script import if possible.
+        imports.addImports(
+                DataColumn.class.getName(),
+                Table.class.getName(),
+                TableFactory.class.getName(),
+                PartitionedTable.class.getName(),
+                PartitionedTableFactory.class.getName(),
+                Array.class.getName(),
+                TypeUtils.class.getName(),
+                ArrayTypeUtils.class.getName(),
+                DateTimeUtils.class.getName(),
+                CompressedString.class.getName(),
```

Comment:
You might as well remove this. It's an old, failed experiment. We should probably let only enterprise have that code.